lib/fetch/index.js

```diff
@@ -1910,8 +1906,8 @@ async function httpNetworkFetch (
 
   // 11. Let pullAlgorithm be an action that resumes the ongoing fetch
   // if it is suspended.
-  const pullAlgorithm = () => {
-    fetchParams.controller.resume()
+  const pullAlgorithm = async () => {
+    await fetchParams.controller.resume()
```

Comment:
While debugging I realized that we call it with an await but as pullAlgorithm is not a async function it basically "decouples" the promise logic.

https://github.com/nodejs/undici/blob/cf4d90dff47dd9d1be9241fe9a324f6b7f671dda/lib/fetch/index.js#L1941

So with this change the await pullAlgorithm will actually have something to await.